src/base/static/components/input-form/__tests__/input-form.test.js

```diff
@@ -64,19 +65,19 @@ describe("InputForm", () => {
     wrapper.setState({
       fields: OrderedMap({
         test1: Map({
-          [constants.FIELD_RENDER_KEY]: 0,
+          [constants.FIELD_RENDER_KEY]: "someCategorytest1",
```

Comment:
I don't think we should be hard-coding `someCategorytest1` here - it's too brittle. I think we should assign variables to `test1` and `someCategory`, and concatenate them in the test itself.